frontend/src/main/components/Review/ReviewTable.js

```diff
@@ -47,8 +47,7 @@ export default function ReviewTable({ reviews, currentUser }) {
         {
             Header: 'comments',
             accessor: 'comments',
-        }
-        
+        } 
```

Comment:
also, your itemID is not being accessed correctly. I think this should be `accessor: 'itemId'`, with a lowercase `d`. 
![image](https://user-images.githubusercontent.com/20908336/201563606-2ee56e2c-7b4b-4d20-a964-0ad814bb946b.png)